magda-web-client/src/AppContainer.js

```diff
@@ -21,20 +21,24 @@ import { Medium } from "./UI/Responsive";
 
 import "./AppContainer.css";
 
+const regex = /(http|https):\/\/(\w+:{0,1}\w*)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%!\-/]))?/;
+
 class AppContainer extends React.Component {
     componentWillMount() {
         this.props.requestWhoAmI();
     }
     renderLink(link) {
-        const regex = /(http|https):\/\/(\w+:{0,1}\w*)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%!\-/]))?/;
-        if (!regex.test(link[1])) {
+        if (link[1].startsWith("mailto")) {
```

Comment:
startsWith is a quite new method (I think it's es2015).
Not sure whether we can use it.
How about I change it to link[1].indexOf("mailto")===0 ?

Reply:
Good catch, fixed.